src/main/java/com/ericsson/ei/waitlist/WaitListStorageHandler.java

```diff
@@ -66,7 +66,7 @@
     private JmesPathInterface jmesPathInterface;
 
     /**
-     * Adds event to the wait-list database if it does not already exists.
+     * Adds event to the wait list database if it does not already exists.
```

Comment:
Same comment here then regarding the spacing. Why not consider it a new word we've invented to describe the concept :)

Reply:
Done